EN/Images/transcripts and raw translations/saber.txt

```diff
@@ -1,21 +1,21 @@
 セイバー
 Saber
 交わらぬはずの並行世界に召喚された謎を追う「騎士王」
-The Knight King who pursues the mystery of why she has been summoned to a parallel world that she is never meant to intersect with
+The King of Knights who pursues the mystery of why she had been summoned to a parallel world she was never meant to intersect with
 
 真名：アルトリア・ペンドラゴン
 後ろに紡い上げた金髪、青と白銀の甲冑が眼早い、凛とした佇まいの女性剣士。彼女こそ、アーサー王伝説として、巷間に名高い、円卓の騎士を束ねる聖剣の騎士である。（生前は性別を偽って活動していたため、アーサー王＝男性として知られている。）
 True Name: Artoria Pendragon
-Saber is a swordswoman possessed of a regal bearing; the blonde hair she has braided behind her head and the blue-silver armor she wears doing much to make her stand out. She is none other than the famous Knight of the Holy Sword, popularly famed as the one who commanded the Round Table in the Arthurian Cycle. (Since she concealed her true gender in life, King Arthur is known as a man)
+Saber is a swordswoman possessed of a regal bearing. Her golden locks and her silver armor, accentuated with blue, make her stick out in the city of Souya. Her True Name is none other than Artoria Pendragon, the celebrated Knight of the Holy Sword, famed as the one who commanded the Round Table during the Arthurian cycle. Because she hid her true gender in life, history remembers her as King Arthur.
 
 死後は、伝説的な英雄として「英霊の座」に刻まれ”サーヴァント”となった。本来、「英霊の座」という仕組みのないこの世界に何故か現界してしまい、その謎を解くため、総耶の街を探索することに……。
-After death, she became a Servant, engraved into the the 'Throne of Heroes' for her legendary deeds. Now she has manifested in a world where the mechanisms of the Throne of Heroes do not exist, and so heads out to explore the city of Souya in order to solve the mystery of her own summoning...
+After death, she became a Servant, engraved into the the 'Throne of Heroes' for her legendary deeds. Now, finding herself manifested in a world where the mechanisms of the Throne of Heroes do not exist, she heads out to explore the city of Souya and uncover the mystery behind her summoning.
 
 生来持つ莫大な魔力により、そのブースト効果で己の身体能力を本来の何倍にも引き上げている。その魔力を集束・加速させた”光”の断層による究極の斬撃[ruby=約束された勝利の剣]「エくスカリバー」[/ruby]や、手に持つ聖剣の姿を不可視にする[ruby=風王結界]「インビシブル・エア」[/ruby]などの宝具を有する。
-Thanks to her innate abundance of magical energy, she can boost her own physical abilities several times above her usual. 
-She holds several Noble Phantasms, such as:
-Excalibur, which is capable of the ultimate slash by focusing and accelerating her magical energy into a rift of light;
-Invisible Air, which renders the Holy Sword in her hands imperceivable.
+Thanks to her innate abundance of magical energy, she can boost her own physical abilities several times beyond her usual output.
+She is equipped with multiples Noble Phantasms:
+Excalibur, the Sword of Promised Victory, capable of unleashing the ultimate slash by focusing and accelerating her magical energy into a rift of light
+Invisible Air, rendering her Holy Sword and thus her identity imperceivable
```

Comment:
Surely it is "**multiple** Noble Phantasms:"?

In this instance we may as well just stick with Invisible Air just making her sword Invisible since it's describing what the effects are.